aea/helpers/multiaddr/crypto.proto

```diff
@@ -0,0 +1,20 @@
+syntax = "proto2";
+
+package crypto.pb;
+
+enum KeyType {
+  RSA = 0;
+  Ed25519 = 1;
+  Secp256k1 = 2;
+  ECDSA = 3;
+}
+
+message PublicKey {
+  required KeyType key_type = 1;
+  required bytes data = 2;
+}
+
+message PrivateKey {
+  required KeyType key_type = 1;
+  required bytes data = 2;
+}
```

Comment:
I don't see that being used; I'd remove

Reply:
Unless you want to have a generic protobuf rep of crypto module; then the proto files should be moved to `aea.crypto`

Reply:
This proto file is the reference one used by libp2p. I kept it as is to avoid having any impact on the serialization algorithm.